test/binary/bad-data-drop-no-data-count.txt

```diff
@@ -0,0 +1,23 @@
+;;; TOOL: run-gen-wasm-bad
+;;; ARGS1: --enable-bulk-memory
+;;; ARGS2: --enable-bulk-memory
```

Comment:
Why not ARGS*?

Reply:
Because this flag doesn't work with ARGS0, unfortunately. We probably should have a less brittle syntax for this.